src/Components/Button.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prop-types
```

Comment:
- [ ] @ZabihullahNooriWardak Great work on your app, however its not good practise disabling linter checks, kindly consider trying to solve the linter error before disabling linters.